Readme.md

```diff
@@ -80,9 +80,17 @@ Use
 After [some setup](docs/Examples.md), using Moya is really simple. You can access an API like this:
 
 ```swift
-provider.request(.Zen) { (data, statusCode, response, error) in
-    if let data = data {
-        // do something with the data
+provider.request(.Zen) { result in
+    switch result {
+    case let .Success(moyaResult):
```

Comment:
`result` and `moyaResult`?  Confusing variable names?  This is what I called these in the Ello app, but I'm open to suggestions for better names!